### ISSUE TITLE
FullPage mode might be partly broken

### ISSUE THREAD

**Issue by TanukiSharp** [openseadragon/openseadragon#2099]
This is actually a question, but also partly an issue I encountered.

In full page mode, I noticed that if I resize the browser, the viewer remains sized to the width and height it had at the moment full page got triggered. The culprit code is here: https://github.com/openseadragon/openseadragon/blob/b1ec71e91cbd722602a1d11c2d0691ca585e03d7/src/viewer.js#L1043-L1044

Any particular reason for it to not be the following instead ?

```js
this.element.style.height = '100%';
this.element.style.width = '100%';
```

The percentage method does not break the current state, as long as using fixed unit, it was probably considered the browser would never be resized, still working the same with percentage. But full page implies "full page all the time", so percentage would be a better approach, in my opinion. Also, it wouldn't be necessary to get the window size beforehand.

For the moment, I solved the problem with CSS, as follow:

```css
.image-viewer.fullpage {
    width: 100% !important;
    height: 100% !important;
}
```

But it's clearly not elegant 😕

**Comment by iangilman** [comment]
Excellent point! Seems like a worthwhile fix, if you're up for making a PR :)

**Comment by iangilman** [comment]
Fixed by #2100.
